src/Missings.jl

```diff
@@ -272,10 +276,17 @@ struct EachReplaceMissing{T, U}
     x::T
     replacement::U
 end
+if VERSION < v"0.7.0-DEV.3309"
 Base.iteratorsize(::Type{<:EachReplaceMissing{T}}) where {T} =
     Base.iteratorsize(T)
 Base.iteratoreltype(::Type{<:EachReplaceMissing{T}}) where {T} =
     Base.iteratoreltype(T)
+else
+Base.IteratorSize(::Type{<:EachReplaceMissing{T}}) where {T} =
```

Comment:
We should add this replacement to Compat.

Reply:
I was going to, but it's weird because `IteratorSize` is defined in 0.6 too.

Reply:
Oh it is? Hm, that is weird.

Reply:
Fredrik is adding it in Compat as `Compat.IteratorSize`, so we could use that if this PR is merged after his is merged and tagged.